README.md

```diff
@@ -598,3 +598,7 @@ Change Log:
 26 October: Added vignette w/ examples for purrr and forcats, Cameron Smith
 
 palmorezm Extended Zhouxin Shi's dplyr filter vignette by adding another example of the filter function's usage and adding detail about the function and its arguments. Data used was identical to that of Zhouxin Shi's create vignette and it was used build on the existing examples. No changes were made to isolate dplyr::filter vigette. As such the read_csv and select functions remain as additional background for the extended portion of this vignette. 
+
+=======
+RickRN Extended the Tidyverse_Vignette.Rmd by Zachary Palmore on Reading Comma Separated Values with "spec_csv" that shows how to Generate a column specification.
+
```

Comment:
Updating the content by adding some source code through some CSS and HTML file residing on foreign servers.